tools/ingest2parquet/Makefile

```diff
@@ -3,22 +3,40 @@ REPOROOT=../../
 
 include $(REPOROOT)/.make.defaults
 
-DOCKER_IMAGE_NAME=ingest2parquet
-DOCKER_IMAGE_VERSION=0.1
+include ../../.make.versions
+
+DOCKER_IMAGE_VERSION=${INGEST_TO_PARQUET_VERSION}
+DOCKER_IMAGE_NAME?=ingest2parquet
 DOCKER_IMAGE=${DOCKER_REGISTRY_ENDPOINT}/$(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_VERSION)
 
 
-venv:	.defaults.lib-src-venv
+venv::	.defaults.lib-src-venv
+
+build::	venv image
+
+test::	test-src test-image
+
+clean:: .defaults.clean
+
+image:: .defaults.lib-src-image
 
-build:	 venv image
+test-src:: .defaults.test-src test-local
 
-test: test-src test-image	
-	@# Help: Test both the source and the docker image
+test-image:: image .defaults.test-image-pytest
 
-clean: .defaults.clean
+test-local::
+	@# Help: Run the *local*.py files in the src directory
+	@source venv/bin/activate;		\
+	export PYTHONPATH=src;			\
+	files=$$(find src -name '*local*.py' );		\
+	for i in $$files; do			\
+	    echo Executing: python $$i;		\
+	    python $$i;				\
+	    if [ $$? -ne 0 ];	then		\
+		exit 1;				\
+	    fi;					\
+	done
 
-image: .defaults.lib-src-image
 
-test-src: .defaults.test-src
 
-test-image: image .defaults.test-image-pytest
+publish:: .transforms.publish
```

Comment:
I think this should be 

`publish::  image .defaults.publish `

as transforms has this
`
.transforms.publish:: .transforms.image .defaults.publish `

Reply:
in fact, how did .transforms.publish work since I don't see an include of .make.transforms.

Reply:
done